webapp/src/app.tsx

```diff
@@ -92,7 +92,9 @@ const App = React.memo((): JSX.Element => {
         dispatch(fetchLanguage())
         dispatch(fetchMe())
         dispatch(fetchClientConfig())
-        history.replace(window.location.pathname.replace((window as any).frontendBaseURL, ''))
+        if (Utils.isFocalboardPlugin()) {
+            history.replace(window.location.pathname.replace((window as any).frontendBaseURL, ''))
+        }
```

Comment:
@harshilsharma63 - Is this line necessary?  I retested this [issue](https://github.com/mattermost/focalboard/issues/1373) by using a focalboard link in a channel message. And this line didn't seem to matter, I tested both ways and the other changes seem to fix the issue.

Reply:
When I tested it it was required. Right now the state of Channels and Boards routers co-existing is a bit messed up, so I'd prefer keeping this.

Making this conditional for plugins only is fine though.